other/appfilter.xml

```diff
@@ -7024,7 +7041,7 @@
     <!-- Life Fits Into Home -->
     <item component="ComponentInfo{tr.gov.saglik.hayatevesigar/com.turktelekom.guvenlekal.ui.activity.SplashActivity}" drawable="life_fits_into_home" />
 
-    <!-- Liftago -->
```

Comment:
can you re-add the > here?

Reply:
me? In my forked appfilter.xml, everything looks fine without missing >